smirnoff_omm/smirnoff.ipynb

```diff
@@ -4,27 +4,35 @@
    "cell_type": "markdown",
```

Comment:
<p>This should be updated to 55.7kcal/mol</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mosdef-hub/mosdef-workflows/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>